app/ui/src/app/integration/import-export/import/integration-import.component.ts

```diff
@@ -57,14 +66,26 @@ export class IntegrationImportComponent implements OnInit, OnDestroy {
     };
   }
 
-  onFileOver(e) {
-    this.hasBaseDropZoneOver = e;
+  onDropFile(): void {
+    this.isDragAndDropImport = true;
+    this.isMultipleImport = this.checkIfMultiple();
+  }
+
+  onDropOverAndOut(event: Event): void {
+    this.checkIfDragAndDrop(event);
+  }
+
+  onFileSelected(): void {
+    this.isDragAndDropImport = false;
+    this.isMultipleImport = this.checkIfMultiple();
   }
 
   ngOnInit() {
-    this.integrationOverviews$ = this.integrationSupportService.watchOverviews();
+    this.integrationOverviews$ = this.integrationSupportService.getOverviews();
```

Comment:
I kinda feel like this component shouldn't need to fetch integration objects at all, at least via these API calls.  If the import endpoint doesn't return the integration objects after importing I think we should change that so it should, which I think will make this component a lot more simple, don't you think?

Reply:
I agree and would have liked to have had that removed before opening the PR, but I'm technically still not done working on the component. If you'd like this resolved before merging I can do that.

Reply:
Yeah, if it's not needed please take it out, especially given that it's making a backend call, we make enough of 'em as it is :-)

Reply:
Sure